examples/example-native-oauth-client/package.json

```diff
@@ -18,15 +18,15 @@
     "@roll-network/auth-sdk": "workspace:*",
     "@roll-network/design-system": "workspace:*",
     "@roll-network/session-manager": "workspace:*",
-    "@wagmi/core": "1.3.6",
+    "@wagmi/core": "1.4.1",
     "@web3-react/core": "^6.0.9",
     "@web3-react/fortmatic-connector": "^6.1.6",
     "@web3-react/injected-connector": "^6.0.7",
     "@web3-react/network-connector": "^6.2.9",
     "@web3-react/portis-connector": "^6.2.11",
     "@web3-react/walletconnect-connector": "^6.2.10",
-    "@web3modal/ethereum": "2.6.2",
-    "@web3modal/react": "2.6.2",
+    "@web3modal/ethereum": "2.7.1",
+    "@web3modal/react": "2.7.1",
```

Comment:
Is it an isolated update to the design-system? Do the other packages relying on the design-system need to be updated as well?

Reply:
Yes, they will need to. These are peer dependencies.

Reply:
Sure, thanks you 
cc: @diegonzs